### DISCUSSION TITLE
Shortcut to constantly keep pressing the left side of the screen for WASD controls?

### DISCUSSION THREAD

**Discussion by heisendelta** [PlayCover/PlayCover#1486] [Q&A] [answered]
Hi. Wuthering waves has a WASD control stick where you have to press the left side of your screen and swipe up, down, left or right to move your character. This is different from games like Genshin where the control pad is in a fixed location. 

Hence the prebuilt WASD keymap doesn't actually move your character if you press one of the keys. If you want to move forward, for instance,  you'd have to press either A or D before pressing W to get your character to move forward. This gets pretty tedious when you're in combat. 

I tried reconfiguring default keybinds using Karabiner (like getting it to trigger a succession of D then W whenever I press W), but that didn't work too well.

I was wondering if there was a way to constantly hold a place on your right screen so you could register movement using only one of the the four WASD keys. 

Thanks in advance!

**Comment by XuYicong** [comment, marked as answer]
Actually playcover has such function, just that it doesn't have a menu for it. Temporarily if you make WASD pad big enough it would switch to the sliding behavior. We'll make a menu/option for that soon.

**Reply by heisendelta** [reply]
Thank you for the reply! Making the pad really big did they trick.

**Reply by c01nd01r** [reply]
Thank you! This advice also works for Tarisland :)
